packages/auth/e2e/auth.e2e.js

```diff
@@ -944,4 +944,110 @@ describe('auth()', () => {
       should.not.exist(resetKeychain);
     });
   });
+
+  describe('validation with signInWithEmailAndPassword', () => {
+    it('it should return an error for signing in an non string based email', () => {
+      (() => {
+        firebase.auth().signInWithEmailAndPassword(0);
+      }).should.throw(
+        '[auth/argument-error] signInWithEmailAndPassword failed: First argument "email" must be a valid string.',
+      );
+    });
+
+    it('it should return an error for signing in an non string based password', () => {
+      (() => {
+        firebase.auth().signInWithEmailAndPassword('test@test.com', 0);
+      }).should.throw(
+        '[auth/argument-error] signInWithEmailAndPassword failed: Second argument "password" must be a valid string.',
+      );
+    });
+
+    it('it should return an error for signing in an empty string based email', () => {
+      new Promise((resolve, reject) => {
```

Comment:
Missing return, so this test will immediately pass while still continuing to run.